apicast/src/apicast.lua

```diff
@@ -65,26 +67,45 @@ function _M:rewrite()
   ngx.ctx.proxy = p
 end
 
-function _M.post_action()
+function _M:post_action()
   local request_id = ngx.var.original_request_id
+  local post_action_proxy = self.post_action_proxy
+
+  if not post_action_proxy then
+    return nil, 'not initialized'
+  end
+
   local p = ngx.ctx.proxy or post_action_proxy[request_id]
 
   post_action_proxy[request_id] = nil
 
   if p then
-    p:post_action()
+    return p:post_action()
   else
     ngx.log(ngx.INFO, 'could not find proxy for request id: ', request_id)
```

Comment:
Yeah I was thinking about that. INFO is first opt-in level so it is not shown by default. 

I guess this should be part of some major log refactoring as there is lots of duplicated / too verbose information already.